portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="UTF-8">
+    <title>STEP Blog</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+
+  <body bgcolor="#ff9999">
```

Comment:
Can this be moved to style.css? 

From checking validity, it seems like the bgcolor attribute is obsolete?
https://google.github.io/styleguide/htmlcssguide.html#HTML_Validity

Same happens to the `<hr color=black>` tag as well.

And if so could you change this for the other files as well? Thanks!

Reply:
Also for completeness, just saw this: https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns